internal/app/api/v1/server.go

```diff
@@ -346,6 +343,56 @@ func (s *TestkubeAPI) InitRoutes() {
 	s.Log.Infow("dashboard uri", "uri", dashboardURI)
 	s.Mux.All("/", proxy.Forward(dashboardURI))
 
+	// set up proxy for the internal GraphQL server
+	s.Mux.All("/graphql", func(ctx *fiber.Ctx) error {
+		header := ctx.Request().Header.Header()
+		ctx.Context().HijackSetNoResponse(true)
+		ctx.Context().Hijack(func(c net.Conn) {
+			// Connect to server
+			serverConn, err := net.Dial("tcp", ":"+s.graphqlPort)
+			if err != nil {
+				s.Log.Errorw("could not connect to GraphQL server as a proxy", "error", err)
+				return
+			}
+
+			// Extract Unix connection
+			serverSock, _ := serverConn.(*net.TCPConn)
+			clientSock, _ := reflect.Indirect(reflect.ValueOf(c)).FieldByName("Conn").Interface().(*net.TCPConn)
+
```

Comment:
I would check for successful type conversion in both cases

Reply:
In other case below code might panic

Reply:
`serverConn` is declared above, while `clientSock` is based on `fiber`'s implementation, so there is no way it could be other type, but I may add the check anyway 👍